jbmc/src/jbmc/jbmc_parse_options.h

```diff
@@ -71,6 +71,7 @@ class optionst;
   "(property):(stop-on-fail)(trace)" \
   "(verbosity):" \
   "(nondet-static)" \
+  "(validate-trace)" \
```

Comment:
Move as HELP and OPT macros into java_trace_validation.h so that it can be reused in other driver programs (e.g. TG)